itests/org.openhab.core.automation.integration.tests/src/main/java/org/openhab/core/automation/integration/test/AutomationIntegrationTest.java

```diff
@@ -356,60 +353,6 @@ public void assertThatARuleWithNonExistingModuleTypeHandlerIsAddedToTheRuleRegis
         assertThat(ruleEngine.getStatusInfo(rule.getUID()).getStatus(), is(RuleStatus.UNINITIALIZED));
     }
 
-    @Test
```

Comment:
I am wondering why you removed these. Don't we need them anymore?

Reply:
For some reason stopping/starting the bundles doesn't work with the bnd test runner so I remove them. We can also add `@Ignore` them for now and see if someone else comes up with a solution some day. :wink:

Reply:
Thanks for clarification. I think it's okay to remove them.